src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -1304,7 +1304,7 @@ public Response syncExternalAccounts(@Context final HttpServletRequest httpServl
     }
 
     @POST
-    @Path("/start_quartz")
+    @Path("/quartz/start")
```

Comment:
I think this change is safe. It's an admin-only endpoint, and we don't automate this at all.